### ISSUE TITLE
Can v-leaflet be used in an intranet environment, where there is no connection to internet ?

### ISSUE THREAD

**Issue by sempervivo** [mstahv/v-leaflet#117]
Can v-leaflet be used in an intranet environment, where there is no connection to internet ? In this case, can it be connected to a geoserver located on the intranet ?

Thank you so much for this addon.

**Comment by willtemperley** [comment]
Yes. Just add e.g. a WMS layer and point it to your Geoserver, wherever that is.

**Comment by sempervivo** [comment]
Great ! thank you very much. Which version should I use with Vaadin 7.2.7 ? Thanks again.

**Comment by sempervivo** [comment]
foiund it: 1.0.0-b1.
Thanks anyway.

**Comment by sempervivo** [comment]
Do you have an example of how to connect to the geoserver. Some simple LTileLayer configuration for an intranet geoserver ?

Thank you.

**Comment by mstahv** [comment]
Try using LWmsLayer, it might be easier to use with basic geoserver setup. It is used for example in [this test code](https://github.com/mstahv/v-leaflet/blob/master/src/test/java/org/vaadin/addon/leaflet/demoandtestapp/LayersTest.java).
